src/pages/documentation/platform-assurance/tutorials/push-debug-view.md

```diff
@@ -0,0 +1,119 @@
+
+If something is not working as expected, there will be an alert with details on how to fix that problem:
+
+![Invalid State](./assets/push-debug-view/push-invalid-state.png)
+
+#### Client Details
+
+This panel checks to see if the device is configured correctly. This includes configuring the extension in Launch, initializing the extension and its prerequisites in your application, and capturing the push token from the device.
+
+If valid, the panel will display the Ecid for the device, the push token, and the Edge Sandbox name and type.
+
+#### Profile Details
+
+Once your client is set up correctly, this panel will check to see if the device is writing to profile. It also validates that the push token in the profile matches the one on the device.
+
+If valid, the panel will show the Ecid for the device, the push token, the app ID of your application, the messaging platform, and whether the push token has been deny listed. The token can be deny listed for various reasons like the user has uninstalled the app or the user has disabled push messaging for the app.
+
+![Blocked](./assets/push-debug-view/push-deny-list-blocked.png)
+
+Finally, at the bottom of the panel is a link that will open this specific profile in a new tab.
+
+#### App Store Credentials & Configuration
+
+This panel validates that the app ID and the messaging platform that was saved in the profile has a matching app surface created. An app surface is where push credentials for the application are uploaded.
+
+If valid, the profile will display the name of the app surface, the app ID, and the name of the messaging service.
+
+Finally, at the bottom of the panel is a link that will open this specific app surface in a new tab.
+
+### Send Test Push
+
+The "Send Test Push" tab can be used to send a test message to your device.
+
+#### Message Composition
+
+There are several panes that can be configured to test different iOS and Android push features. Once configured, press the "Send Test Push Notification" button to send your message.
+
+![Send Push](./assets/push-debug-view/push-send.png)
+
+##### Message
+
+In the "Message" pane, a title and body for the message can be provided. The silent notification feature can be enabled here as well.
+
+![Message Pane](./assets/push-debug-view/push-message-pane.png)
+
+##### Push Target
+
+The "Push Target" pane allows you to customize which push token and app surface to use when sending the push message.
+
+This information will be provided by default if the "Validate Setup" tab is showing three green checkmarks. But you may provide your own push token and app surface at any time, even if your app isn't fully configured.
```

Comment:
The final sentence is a bit too informal. I'd go with - "However, you can provide your own push token and app surface, even if your app isn't fully configured."